test

```diff
@@ -28,7 +28,16 @@ fi
 echo "Checking govet..."
 go vet $PKG_VET
 
-echo "Running tests..."
-go test -timeout 60s -cover $@ ${PKG} --race
+if [ "${ACTION:-TEST}" != "COMPILE" ]; then
+    echo "Running tests..."
+    go test -timeout 60s -cover $@ ${PKG} --race
+else
+    PKG=$(cd gopath/src/${REPO_PATH}; go list ./... | \
+    	grep --invert-match vendor)
+    echo "Compiling tests..."
```

Comment:
Running go tests, compiling is misleading, implying you're producing the test binaries